zone/aa.cpp

```diff
@@ -53,9 +53,11 @@ void Mob::TemporaryPets(uint16 spell_id, Mob *targ, const char *name_override, u
 	// yep, even these need pet power!
 	int act_power = 0;
 
-	if (IsClient()) {
-		act_power = CastToClient()->GetFocusEffect(focusPetPower, spell_id);
-		act_power = CastToClient()->mod_pet_power(act_power, spell_id);
+	if (IsOfClientBot()) {
+		act_power = GetFocusEffect(focusPetPower, spell_id);
+		if (IsClient()) {
+			act_power = CastToClient()->mod_pet_power(act_power, spell_id);
```

Comment:
This doesn't seem to do anything.

Reply:
I can always remove the mod_pet_power, but are any of the mod_functions actually doing anything?
Might be best to remove those as part of another PR.

Reply:
I don't see anything using it, making another PR to remove it seems needless.

Reply:
Doesn't broaden scope of this much to remove a method only used in `6` places in `5` files.
![image](https://user-images.githubusercontent.com/89047260/217424584-f0c90e23-e368-4184-825c-1f2424780617.png)

Reply:
Refactors or removals shouldn't really be tucked into another PR in general

If we have to back out these changes for whatever reason they're now coupled.

`mod_` back in the day were "custom stubs" so people could write their own hooks in isolation easily enough

This is before we had the option to do lua modding which hasn't picked up a ton of steam

Reply:
Yeah, makes sense for something bigger than a 6 line change.

Reply:
I raised #2855 as a reminder that mod_functions can likely be cleaned up/removed. I will not be touching them as part of this PR, as it has nothing to do with the scope of this PR.

Reply:
https://github.com/EQEmu/Server/pull/2856